### PR TITLE
Move Organisation translation index page to the GOV.UK Design System

### PR DESCRIPTION
**Pull request by davidgisbey**
## Description

This PR moves the organisation translation index page to the GOV.UK Design System

- duplicates the controller tests
- adds the organisation translation index page
- adds confirm destroy endpoint

## Screenshots

image

image

## Trello card

https://trello.com/c/JdVibxnr/274-organisation-translations-index-page

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

**Review by farahTW — APPROVED** [review]

File: app/views/admin/organisation_translations/confirm_destroy.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<% content_for :context, @organisation.name %>
+<% content_for :page_title, "Delete #{@translation_locale.native_and_english_language_name} translation" %>
+<% content_for :title, "Delete translation" %>
+<% content_for :title_margin_bottom, 6 %>
+
+<div class="govuk-grid-row">
+  <section class="govuk-grid-column-two-thirds">
+    <%= form_with url: admin_person_translation_path(@organisation, @translation_locale), method: :delete do %>
```
Comment (chriswhitehouse-tw) [inline app/views/admin/organisation_translations/confirm_destroy.html.erb:8]: this needs to be 'admin_organisation_translation_path'

**Review by chriswhitehouse-tw — CHANGES_REQUESTED** [review]
Hi David,

Generally looks good, apart from the organisation translation confirm delete for url.

Also, although the changes to the person translation page, look appropriate, its a bit confusing have the commit in this PR. Perhaps is should be moved to its own PR?

**Review by davidtrussler — APPROVED** [review]
LGTM 👍

File: app/views/admin/organisation_translations/confirm_destroy.html.erb (hunk 1 above)
Comment (chriswhitehouse-tw) [reply app/views/admin/organisation_translations/confirm_destroy.html.erb]: updated

**Review by chriswhitehouse-tw — APPROVED** [review]

**Pull request merged**